ui/tailwind.config.ts

```diff
@@ -4,7 +4,7 @@ import type { Config } from 'tailwindcss'
 const config: Config = {
   content: [
     './index.html',
-    './**/*.{vue,js,ts,jsx,tsx}',
+    './src/**/*.vue',
```

Comment:
This is _probably ok_ so long as any classes we're using in `.ts` files are explicitly included in the bundle - we may not be doing that much at this project's level though

Reply:
Yeah I just followed what we had in cloud. Can't imaging we're doing anything here that we're not doing in cloud.

Reply:
good point